client/client_interfaces.go

```diff
@@ -215,7 +215,7 @@ type SecretAPIClient interface {
 
 // ConfigAPIClient defines API client methods for configs
 type ConfigAPIClient interface {
-	ConfigList(ctx context.Context, options swarm.ConfigListOptions) ([]swarm.Config, error)
+	ConfigList(ctx context.Context, options SwarmConfigListOptions) ([]swarm.Config, error)
```

Comment:
I haven't looked closely if it will be ambiguous anywhere by I think for these we may be able to drop the `Swarm` prefix; they were all clobbered together under swarm, but everywhere else we have "config" "secrets", "nodes".